gulpfile.js

```diff
@@ -15,27 +15,13 @@ const gulp = require('gulp');
 const rename = require('gulp-rename');
 const cp = require('child_process');
 
-const microprofileServerName = 'org.eclipse.lsp4mp.ls-uber.jar';
-const microprofileServerDir = '../lsp4mp/microprofile.ls/org.eclipse.lsp4mp.ls';
-
 const quarkusServerExtGlob = 'com.redhat.quarkus.ls!(*-sources).jar';
 const quarkusServerExtDir = '../quarkus-ls/quarkus.ls.ext/com.redhat.quarkus.ls'
 
-const microprofileExtensionDir = '../lsp4mp/microprofile.jdt';
-const microprofileExtension = 'org.eclipse.lsp4mp.jdt.core';
-
 const quarkusExtensionDir = '../quarkus-ls/quarkus.jdt.ext';
 const quarkusExtension = 'com.redhat.microprofile.jdt.quarkus';
 
-
-gulp.task('buildMicroProfileServer', (done) => {
-  cp.execSync(mvnw() + ' clean install -DskipTests', { cwd: microprofileServerDir , stdio: 'inherit' });
-  gulp.src(microprofileServerDir + '/target/' + microprofileServerName)
-    .pipe(gulp.dest('./server'));
-  done();
-});
-
-gulp.task('buildQuarkusServerExt', (done) => {
+gulp.task('buildServer', (done) => {
   cp.execSync(mvnw() + ' clean verify -DskipTests', { cwd: quarkusServerExtDir , stdio: 'inherit' });
   gulp.src(quarkusServerExtDir + '/target/' + quarkusServerExtGlob)
```

Comment:
.pipe(rename('com.redhat.quarkus.ls.jar'))

Reply:
I follow this idea but using const, fixed.